components/prospects/create-prospect-note.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react'
+import {
+  Box,
+  Button,
+  TextField,
+  Typography
+} from '@mui/material'
+import CancelIcon from '@mui/icons-material/Cancel'
+import CreateIcon from '@mui/icons-material/Create'
+import SaveIcon from '@mui/icons-material/Save'
+
+function CreateProspectNote({ prospect }) {
+  const [prospectNote, setProspectNote] = useState('')
+  const [isEditing, setIsEditing] = useState(false)
+
+  const handleProspectNoteChange = (event) => {
+    setProspectNote(event.target.value)
+  }
+
+  const handleCreateNote = () => {
```

Comment:
Can you get rid of this function? If a function does less the one single command, then you probably don't need a function.